src/reducers/authReducer.ts

```diff
@@ -19,9 +21,17 @@ export const updateLastRenewAt = {
   type: UPDATE_LAST_RENEW_AT
 };
 
+const SET_USER_GROUPS = 'AUTH_REDUCER_SET_USER_GROUPS';
+export const setUserGroups = (groups: Array<string>) => ({
+  type: SET_USER_GROUPS,
+  payload: { groups }
+});
+
 const initialState: authReducerState = {
   lastActiveAt: newTimeStamp(),
-  lastRenewAt: newTimeStamp()
+  lastRenewAt: newTimeStamp(),
+  groups: [],
+  userGroupsSet: false
```

Comment:
This helps us figure out if the array is empty because we haven't loaded the info yet, or because the user doesn't have any groups.